frontend/src/metabase/parameters/utils/parameter-values.unit.spec.js

```diff
@@ -186,6 +186,23 @@ describe("parameters/utils/parameter-values", () => {
       ).toEqual([123.456]);
     });
 
+    it("should parse parameter value that is a comma-separated list of numbers", () => {
+      const numberParameter = {
+        id: 111,
+        slug: "numberParameter",
+        type: "number/=",
+      };
+      expect(
+        getParameterValueFromQueryParams(
+          numberParameter,
+          {
+            [numberParameter.slug]: "1, 2,   3",
```

Comment:
There's really no difference between `  2` and `     3`. If you handle spaces in front of a number, this type of test should not give you any more certainty.

It's better to check for both spaces before and after the number. E.g:
`1,2   ,      3`

Reply:
Other scenarios that we're not testing:

1. `,,,` returns `Nan`
2. ` ` returns `Nan` 
3. `1,,2,3,4` returns `1` (drops everything after the comma)
4. `1, ,2,3,4` returns `1` (drops everything after the space)
5. `1,2,3,` (trailing comma) returns `1`
6. `1,0,000,2` returns `1,0,0,2` (correct)
7. `0,000,1,2` returns `0,0,1,2` (correct)


There's probably more, but this is just from the top of my head.

Reply:
@nemanjaglumac Thanks, added these cases.

Reply:
There's no end to potential cases here.

I had trouble falling asleep (took me 3 instead of the usual 1 minute) last night thinking about how, in my native language, the american dot notation for decimals is actually comma notation. And so on and so forth.

For psychiatric prevention reasons, it may be worth mentioning that a run of the question will quickly tell the user if the filter is filled with invalid parameters.